privacy-on-beam/pbeam/count.go

```diff
@@ -107,16 +122,39 @@ func Count(s beam.Scope, pcol PrivatePCollection, params CountParams) beam.PColl
 		newDecodePairInt64Fn(partitionT.Type()),
 		countPairs,
 		beam.TypeDefinition{Var: beam.XType, T: partitionT.Type()})
+	return countWithCorrectPartitions(s, partitions, epsilon, delta, maxPartitionsContributed,
+		params, noiseKind, countsKV)
+
+}
+
+func countWithCorrectPartitions(s beam.Scope, partitions []beam.PCollection, epsilon, delta float64,
+	maxPartitionsContributed int64, params CountParams, noiseKind noise.Kind, countsKV beam.PCollection) beam.PCollection {
+	if len(partitions) == 0 { // No partitions specified.
```

Comment:
Is it possible to do this check in `count` and only call `countWithCorrectPartitions` if len > 0? Seems a little cleaner that way.